pkg/gui/view_helpers.go

```diff
@@ -111,7 +111,11 @@ func (gui *Gui) returnFocus(g *gocui.Gui, v *gocui.View) error {
 		// always fall back to services view if there's no 'previous' view stored
 		previousView, err = g.View("services")
 		if err != nil {
-			gui.Log.Error(err)
+			// Or fall back to the containers view if we are not in a docker-compose project
```

Comment:
We can instead use gui.initiallyFocusedViewName() here and go 
```
previousView, err = g.View(gui.initiallyFocusedViewName())
```

Reply:
That seems like a better idea, I was searching for something like that but could not find it.

Reply:
Fixed it